Sources/FacebookCoreTests/AccessTokenCacheTests.swift

```diff
@@ -20,12 +20,36 @@
 import XCTest
 
 class AccessTokenCacheTests: XCTestCase {
-  func testHasClearCacheMethod() {
-    let cache = AccessTokenCache()
+  var store = FakeSecureStore()
+  lazy var cache = {
+    AccessTokenCache(secureStore: self.store)
+  }()
+
+  func testAccessTokenCaching() {
     cache.accessToken = AccessTokenFixtures.validToken
+    XCTAssertEqual(cache.accessToken, AccessTokenFixtures.validToken,
+                   "The cached access token should equal the passed in token")
 
-    cache.clearCache()
+    cache.accessToken = nil
+    XCTAssertNil(cache.accessToken,
+                 "The cached token should be nil after setting to nil")
+  }
 
-    // TODO: add assertion for clearing token when caching logic is implemented
+  func testAccessTokenCacheErrorHandling() {
+    store.alwaysError = true
+
+    cache.accessToken = AccessTokenFixtures.validToken
+    XCTAssertNil(cache.accessToken,
+                 "When `alwaysError == true`, cache should be nil")
```

Comment:
nit: This is more describing the test fixture behavior than the production code behavior. I would probably have gone with something like, "Should not cache an access token that cannot be persisted in the store"